tests/integration/annotation_import/test_label_import.py

```diff
@@ -110,3 +113,42 @@ def test_wait_till_done(client, configured_project, predictions):
         input_annot['uuid'] for input_annot in label_import.statuses
     ]
     assert set(input_uuids) == set(status_uuids)
+
+
+@pytest.mark.parametrize('data_type_class', [
+    AudioData, ConversationData, DicomData, DocumentData, HTMLData, ImageData,
+    TextData
+])
+def test_import_data_types(client, configured_project,
+                           data_row_json_by_data_type, annotations_by_data_type,
+                           data_type_class):
+
+    project_id = configured_project.uid
+
+    data_type_string = data_type_class.__name__[:-4].lower()
+    data_row_ndjson = data_row_json_by_data_type[data_type_string]
+    dataset = next(configured_project.datasets())
+    data_row = dataset.create_data_row(data_row_ndjson)
+
+    annotations_ndjson = annotations_by_data_type[data_type_string]
+    annotations_list = [
+        label.annotations
+        for label in NDJsonConverter.deserialize(annotations_ndjson)
+    ]
+    labels = [
+        lb_types.Label(data=data_type_class(uid=data_row.uid),
+                       annotations=annotations)
+        for annotations in annotations_list
+    ]
+
+    label_import = LabelImport.create_from_objects(
+        client, project_id, f'test-import-{data_type_string}', labels)
+    label_import.wait_until_done()
+
+    assert label_import.state == AnnotationImportState.FINISHED
+    assert len(label_import.errors) == 0
+    exported_labels = configured_project.export_labels(download=True)
+    objects = exported_labels[0]['Label']['objects']
+    classifications = exported_labels[0]['Label']['classifications']
+    assert len(objects) + len(classifications) == len(labels)
+    DataRow.bulk_delete([data_row])
```

Comment:
any reason for using `DataRow.bulk_delete` over `data_row.delete()`?

Reply:
I can't find the `data_row.delete()` method does it exist?